app/controllers/chatrooms_controller.rb

```diff
@@ -0,0 +1,74 @@
+class ChatroomsController < ApplicationController
```

Comment:
1. Commit message is about "Create database". Why you created the controller 
2. Are you using all of the actions in this pull request? Please remove the actions that are not using. 
3. Remove all of the comments. 
4. Remove the scaffold-ed views that are not related to this pull request.

Reply:
I'm using code of pull request task "init project", so more than 15 file changed. 
The code must have 'init project', then the application can run @@

Reply:
I'm using command 'scaffold' , so it created controller

Reply:
Ok. I'm fine @mdmahmudurrahman

Reply:
I'm removed file 'scaffold' created